plugins/password.py

```diff
@@ -52,14 +51,14 @@ def password(text, notice):
 
     # defaults to lowercase alpha + numbers password if the okay list is empty
     if not okay:
-        okay = list(string.ascii_lowercase) + list(string.digits)
+        okay = string.ascii_lowercase + string.digits
```

Comment:
This should probably be `okay += string.ascii_lowercase + string.digits` or `okay = list(string.ascii_lowercase + string.digits)` otherwise gen.shuffle() will throw a TypeError exception.

`TypeError: 'str' object does not support item assignment`